src/test/resources/template/registerremotemodel-createindex.json

```diff
@@ -0,0 +1,84 @@
+{
+    "name": "semantic search with local pretrained model",
+    "description": "Setting up semantic search, with a local pretrained embedding model",
+    "use_case": "SEMANTIC_SEARCH",
+    "version": {
```

Comment:
just wondering what happens on reprovisioning when we change version or the other fields not in workflows?

Reply:
This will fail with message `Template does not contain any modifications`. We only check for modifications within the workflow nodes